openlibrary/utils/solr.py

```diff
@@ -77,6 +77,16 @@ def get_many(
         ).json()
         return [doc_wrapper(doc) for doc in resp['response']['docs']]
 
+    def update_in_place(
+        self,
+        request,
+    ):
+        resp = requests.post(
+            f'{self.base_url}/update?update.partial.requireInPlace=true,commit=true',
+            json=request,
+        ).json()
+        return resp
```

Comment:
I would return the whole thing and drop the variable definition since we don't use it, but finicky